ctrans.py

```diff
@@ -318,12 +371,19 @@ def is_script(filename):
                 autodetect = True
         if opt == '-o':
             decodeas = arg
-            
+        if opt == '--t':
+            trace = True
+        if opt == '--srclang':
+            src_lang = arg
+        if opt == '--tolang':
+            lang = arg
+        if opt == '--ext':
+           ext = arg
     
     if dir_mode:
         scan_dir(target)
     else:
         scan_file(target)
             
```

Comment:
I know this was an issue in the original source code, not your patch, but can you remove all but the last newline?